core/web/resolver/node.go

```diff
@@ -29,7 +29,7 @@ func NewNodes(nodes []types.Node) []*NodeResolver {
 
 // ID resolves the node's unique identifier.
 func (r *NodeResolver) ID() graphql.ID {
-	return int32GQLID(r.node.ID)
+	return graphql.ID(r.node.Name) //TODO change on UI side too?
```

Comment:
This may require changes in the UI, depending on what's being done there with this value.

Reply:
We can optionally drop the display column since it is redundant now, but the rest I think I will be able to patch up.
![Screenshot from 2022-10-03 07-40-19](https://user-images.githubusercontent.com/1194128/193580951-f5525428-a6f0-4380-b461-f0629eb14bd9.png)